src/bootstrap/bootstrap.py

```diff
@@ -315,7 +315,7 @@ def build_triple(self):
         try:
             ostype = subprocess.check_output(['uname', '-s']).strip().decode(default_encoding)
             cputype = subprocess.check_output(['uname', '-m']).strip().decode(default_encoding)
-        except (subprocess.CalledProcessError, WindowsError):
+        except subprocess.CalledProcessError:
```

Comment:
It can remain the same if you wish, it does not break Solaris (which is how I missed it the first time).